pandas/tests/io/test_packers.py

```diff
@@ -1,31 +1,27 @@
-import pytest
-
-from warnings import catch_warnings
-import os
 import datetime
+from distutils.version import LooseVersion
 import glob
+import os
+from warnings import catch_warnings
```

Comment:
can u remove this file from setup.cfg (as u have i sorted it)

Reply:
done